X/Xorg_libxcb/build_tarballs.jl

```diff
@@ -59,7 +59,7 @@ dependencies = [
     BuildDependency("Xorg_util_macros_jll"),
     BuildDependency("Xorg_xproto_jll"),
     BuildDependency("Xorg_xcb_proto_jll"),
-    Dependency("XSLT_jll"),
+    Dependency("XSLT_jll"),  # remove in next release --- only used for docs check: https://bugs.freedesktop.org/show_bug.cgi?id=23863
```

Comment:
You can already remove it now.  It'll be still a dependency of the package, but won't be loaded.

Reply:
Made a comment faster than I could ask my clarifying question :-D I'll remove and push again.